addons/website/static/src/js/text_processing.js

```diff
@@ -5,9 +5,6 @@ import * as OdooEditorLib from "@web_editor/js/editor/odoo-editor/src/utils/util
 // SVG generator: contains all information needed to draw highlight SVGs
 // according to text dimensions, highlight style,...
 const _textHighlightFactory = {
-    underline: targetEl => {
-        return drawPath(targetEl, {mode: "line"});
-    },
```

Comment:
With this commit as it is, if a current 17.0 user did use that effect, it will now crash on every page load... we of course cannot merge that in 17.0 at this point :grimacing: 

1. Let's move this commit for master as well, it's too late for 17.0
2. We now have to consider migration / compatibility

Reply:
Will be done in https://github.com/odoo/odoo/pull/149820 :+1: